### DISCUSSION TITLE
Error "java.lang.ClassNotFoundException"

### DISCUSSION THREAD

**Discussion by Dom9988** [danthe1st/Mee6-bypasser#52] [Support]
Last step of "Running" while building the bot from cloned project. Any Help?
(i'm not too good with coding at all, sorry)

**Comment by noobidev** [comment]
Can you please provide the full stack trace?

**Reply by Dom9988** [reply]
Idk how to translate because i'm Italian :') btw here's a screen: 
![Cattura](https://user-images.githubusercontent.com/89169337/130305504-3631841d-ea6d-483f-9c15-2cd6fc8be4e1.PNG)

**Reply by danthe1st** [reply]
What is the output of `java -version`? If you rename the file `mee-bypasser.jar` to `mee-bypasser.zip`, does it contain a file named `MeeBypasser.class` in a directory `io/github/mee/bypasser`? Is that file present in `target/classes`?
